RecoEgamma/EgammaIsolationAlgos/python/pfClusterIsolation_cfi.py

```diff
@@ -50,9 +54,14 @@
                                                       energyEndcap = cms.double(0)
                                                       )
 
+ootPhotonHcalPFClusterIsolationProducer = photonHcalPFClusterIsolationProducer.clone()
+ootPhotonHcalPFClusterIsolationProducer.candidateProducer = cms.InputTag('ootPhotons')
```

Comment:
it looks like corresponding edits in reducedEgamma are missing for these updates.